chain-addr/src/lib.rs

```diff
@@ -54,7 +54,7 @@ cfg_if! {
 // Allow to differentiate between address in
 // production and testing setting, so that
 // one type of address is not used in another setting.
-#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+#[derive(Debug, Clone, Copy, PartialEq, PartialOrd, Eq, Hash)]
```

Comment:
Why are these `PartialOrd` impls needed? It's unusual to add ordering to enums that don't have a business logic order.